internal/redis/dump/dump.go

```diff
@@ -26,12 +27,15 @@ func (f *dump) Apply(ctx *sqlite.Context, values ...sqlite.Value) {
 
 	result, err := f.rdb.Dump(context.TODO(), key).Result()
 	if err != nil {
+		if errors.Is(err, redis.Nil) {
+			ctx.ResultNull()
+			return
+		}
 		ctx.ResultError(err)
 		return
 	}
 
-	ctx.ResultText(result)
-
+	ctx.ResultText(fmt.Sprintf("%+q", result))
```

Comment:
From: https://blog.golang.org/strings, to match output described here: https://redis.io/commands/dump